components/Base/input/index.vue

```diff
@@ -159,6 +169,9 @@ export default {
         return 'text'
       }
       return this.type
+    },
+    showErrorMsg () {
+      return typeof (this.errorMessage === 'string' && !!this.errorMessage.length)
     }
```

Comment:
ask:

why this method returns `typeof (boolean && boolean)` @doohanas  ? this methods actually return a string with a value of `'boolean'` right?

Reply:
I expected to create this method to get a value of 'boolean' and yes,  it will return string a string with a value of 'boolean'. I guess, it's better to write like this ` return this.errorMessage === 'string' && !!this.errorMessage.length` kak @Ibwedagama?

Reply:
in my personal opinion, I think it's better to stay consistent with the function name and what value it's actually produced. 

In this case, the method name is `showErrorMsg` so I thought it will return a `True` or `False` value and not a string. Yes in my opinion it's better to just return: 

```js
return this.errorMessage === 'string' && !!this.errorMessage.length
```

Reply:
Yes, I agree with that if I try to stay consistent with the function name and what value it's actually produced. So, it's a good insight for me. Thanks kak @Ibwedagama.

Reply:
I'm sorry @doohanas after looking closely to this function actually the code should be like this:

```js
return typeof this.errorMessage === 'string' && !!this.errorMessage.length

// check if this.errorMessage value is a string and it has a value
```

✌🏼 😆

Reply:
Okay then, [this ](https://github.com/jabardigitalservice/dedi-landing-page/pull/77/commits/8b08482deb16afb7ef6562a3723cdb29b86f7c02)a changes kak 👍🏽